Week4/Assignment4/app/src/main/java/com/raywenderlich/bootcamp/blockbusterrecyclerview/model/MoviesDataSource.kt

```diff
@@ -0,0 +1,61 @@
+package com.raywenderlich.bootcamp.blockbusterrecyclerview.model
```

Comment:
I like how you encapsulated the data so this can later be changed out and provided some functions that could later be refactored into an interface like the lessons we're seeing this week.